### ISSUE TITLE
Create variable code action not working for remote method calls

### ISSUE THREAD

**Issue by pubudu91** [ballerina-platform/ballerina-lang#30763]
Consider the following:
```ballerina
http:Client ep = check new("https://localhost::9090");
ep->get("/foo");
```
The create variable code action doesn't get suggested for the above remote method call.

**Comment by IMS94** [comment]
At `ep->get("/foo");`, the return type is interred based on the type at LHS. Therefore the implementation is not straightforward.

**Comment by IMS94** [comment]
Same as https://github.com/ballerina-platform/ballerina-lang/issues/36347. This was implemented with #38267
